### PR TITLE
Modify the root README and CONTRIBUTING instructions to be more clear about the need for a temporary Maven repository

### PR DESCRIPTION
**Pull request by sgilda**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,45 +110,8 @@ General Guidelines
 
 * The `<artifactId>` in the quickstart `pom.xml` file should follow the template: `jboss-<target-product>-<quickstart-name>`. For example, the `<artifactId>` for the `greeter` quickstart in the EAP project is `jboss-greeter`. The `<artifactId>` for `errors` quickstart in the Fuse project is `jboss-fuse-errors`.
 
-* The JBoss developer Maven repository that contains newly staged artifacts is located at [developer.github.io](http://jboss-developer.github.io/temp-maven-repo/).
+* The JBoss developer Maven repository, which contains newly staged artifacts, is located at [developer.github.io](http://jboss-developer.github.io/temp-maven-repo/). See [Configure Maven](#configure-maven) below for instructions how to configure your settings to use this repository.
```
Comment (rafabene) [inline CONTRIBUTING.md]: 'is located at [jboss-developer.github.io]' instead of 'is located at [developer.github.io]'

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -224,6 +187,50 @@ To setup the environment you need to follow these steps.
 
             sudo easy_install pygments
 
+Configure Maven
+---------------
+
+If you are working with quickstarts currently under development in the master branch, you need access to artifacts currently under development. The JBoss developer Maven repository, which contains newly staged artifacts, is located at [developer.github.io](http://jboss-developer.github.io/temp-maven-repo/).
```
Comment (rafabene) [inline CONTRIBUTING.md]: same here

**Comment by sgilda** [comment]
@rafabene : Good catch! It's been that way from the beginning! Is this good now?

**Comment by rafabene** [comment]
That's fine for me now :) Thanks

**Pull request merged**